plugins/experimental/maxmind_acl/CMakeLists.txt

```diff
@@ -14,11 +14,16 @@
 #  the License.
 #
 #######################
+find_package(maxminddb QUIET)
```

Comment:
@traeak observed that `QUIET` silences the messages indicating the package was found. We probably don't want that. 
```cmake
find_package(maxminddb)
```
will keep the informational messages.

We want to change this to an `auto_option` but we can change this and `stek_share` together in a separate PR.